cmd/minikube/cmd/status.go

```diff
@@ -367,6 +370,13 @@ func nodeStatus(api libmachine.API, cc config.ClusterConfig, n config.Node) (*St
 	stk := kverify.ServiceStatus(cr, "kubelet")
 	st.Kubelet = stk.String()
 
+	if cc.ScheduledStop != nil {
+		timeToStop := time.Now().Add(cc.ScheduledStop.Duration)
```

Comment:
When a user runs:
```
minikube stop --schedule 2m
```

then `cc.ScheduledStop.InitiationTime` and `cc.ScheduledStop.Duration` is stored in cc.ScheduledStop

We want to compare the duration to that initiation time, not `time.Now()`, because `minikube status` will have been run after. 

I would rewrite this as:

```go
if cc.ScheduledStop != nil {
  st.TimeToStop = time.Until(cc.ScheduledStop.Initiation.Add(cc.ScheduledStop.Duration))
} else {
  st.TimeToStop = Irrelevant
}
```

Reply:
fixed it